rollup.config.js

```diff
@@ -42,7 +43,7 @@ export default {
 	plugins: [
 		replace({
 			preventAssignment: true,
-			'process.env.dev': !production,
+			'process.env.dev': vercelEnv !== undefined ? vercelEnv !== "production" : !production,
```

Comment:
I am not sure exactly why you are comparing `vercelEnv !== "production"`. How about?
`vercelEnv || !production`

Reply:
vercelEnv can be production, preview or development, but it's undefinded on localhost

Reference: https://vercel.com/docs/environment-variables#environments

Reply:
but it can be `vercelEnv || !production`

Reply:
process.env.dev is a boolean not a string, vercelEnv is a string or undefined